src/main/scala/beam/utils/GpxCompareSimulationVsReference.scala

```diff
@@ -51,7 +51,7 @@ object GpxCompareSimulationVsReference {
               dstid.foreach {
                 destination =>
                   {
-                    outWriter.write(s"""<rte>""")
+                    outWriter.write("""<rte>""")
```

Comment:
You can also get rid of the triple quotes.

Reply:
done